src/test/clojure/xtdb/operator/table_test.clj

```diff
@@ -112,7 +112,7 @@
                         {:params {'?x53 "RAIL"}}))))
 
 (t/deftest test-date-subtraction-bug-435
```

Comment:
Not sure on the exact issue/bug this is referencing (couldnt find an issue #435) - possible this test is redundant now given that we're not using durations? @jarohen

Reply:
xtdb/core2#435 - this was pre-migration :slightly_smiling_face: